create_package.py

```diff
@@ -207,27 +211,33 @@ def update_client_version(logger):
         CLIENT_ROOT, ADDON_CLIENT_DIR, "version.py"
     )
     if not os.path.exists(version_path):
-        logger.debug("Creating version.py in client directory")
+        log.debug("Creating version.py in client directory")
 
-    logger.info("Updating client version")
-    with open(version_path, "w") as stream:
-        stream.write(VERSION_PY_CONTENT)
+    log.info("Updating client version")
+    Path(version_path).write_text(VERSION_PY_CONTENT, encoding="utf-8")
 
 
-def build_frontend():
+def build_frontend() -> None:
+    """Build frontend code using yarn.
+
+    Raises:
+        RuntimeError: If yarn executable was not found or frontend
+            build failed.
+
+    """
     yarn_executable = _get_yarn_executable()
     if yarn_executable is None:
-        raise RuntimeError("Yarn executable was not found.")
+        msg = "Yarn executable was not found."
```

Comment:
Why?

Reply:
https://docs.astral.sh/ruff/rules/raw-string-in-exception/